packages/core/src/read/impl.js

```diff
@@ -207,8 +207,6 @@ export async function readRefresh(context, action) {
         readValue: response.value,
       })
     } else {
-      error(`Error while refreshing (read) ${uri} `, response)
```

Comment:
Hmm... Removing this would cause real errors in reads (custom read functions) to be silently ignored.... Perhaps it i better to have it at info (log) level.